policyengine_canada/variables/gov/provinces/nu/tax/income/credits/nu_married_status_credit_eligible.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_canada.model_api import *
+
+
+class nu_married_status_credit_eligible(Variable):
+    value_type = bool
+    entity = Household
+    label = "Eligible for the Nunavut married status credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.NU
+
+    def formula(household, period, parameters):
+        person = household.members
+        p = parameters(
+            period
+        ).gov.provinces.nu.tax.income.credits.eligible_dependent_credit
+        spouse = person("is_spouse", period)
+        dependent = person("is_dependant", period)
+        dependent_spouse = household.any(spouse & dependent)
+        dependent_non_spouse = household.any((~spouse) & dependent)
+        return dependent_spouse & (~dependent_non_spouse)
```

Comment:
a household has to have **both** a spouse dependent and a not dependent non spouse? wouldn't that be the head?